engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/ProcessInstanceRestServiceInteractionTest.java

```diff
@@ -4551,4 +4554,75 @@ private void verifyTaskComments(List<Comment> mockTaskComments, Response respons
     assertEquals(mockComment.getTime(), returnedTime);
     assertEquals(mockComment.getFullMessage(), returnedFullMessage);
   }
+
+  @Test
+  public void testWithVariablesInReturnSetToFalseGetProcessList() {
+    List<ProcessInstance> mockProcessInstanceList = new ArrayList<>();
+    ProcessInstance mockInstance = MockProvider.createMockInstance();
+    mockProcessInstanceList.add(mockInstance);
+		
+    ProcessInstanceQuery sampleInstanceQuery = mock(ProcessInstanceQuery.class);
+			
+    MockedStatic<QueryUtil> mockedStatic = mockStatic(QueryUtil.class);
+    Mockito.when(QueryUtil.list(sampleInstanceQuery, 0, 1)).thenReturn(mockProcessInstanceList);
+    assertEquals(1, QueryUtil.list(sampleInstanceQuery, 0, 1).size());
+
+    when(runtimeServiceMock.createProcessInstanceQuery()).thenReturn(sampleInstanceQuery);
+    when(sampleInstanceQuery.processInstanceId(anyString())).thenReturn(sampleInstanceQuery);
+    when(sampleInstanceQuery.listPage(0, 1)).thenReturn(mockProcessInstanceList);
+	         
+    VariableMap mockInstanceWithVariable = MockProvider.createMockSerializedVariables();
+    when(runtimeServiceMock.getVariables(anyString())).thenReturn(mockInstanceWithVariable);
+
+    given()
+      .queryParam("withVariablesInReturn", false)
```

Comment:
The default behaviour is that  `withVariablesInReturn` is not part of the request. If `withVariablesInReturn` is not present then it is the same sa `withVariablesInReturn` is `false`